README.md

```diff
@@ -486,6 +486,17 @@ See the `adapters/` directory for bindings to *libevent* and a range of other ev
 
 ## Other details
 
+### Cluster node iterator
+
+A `redisClusterNodeIterator` can be used to iterate on all known master nodes in a cluster context.
+First it needs to be initiated using `redisClusterInitNodeIterator()` and then you can repeatedly
+call `redisClusterNodeNext()` to get the next node from the iterator.
```

Comment:
Should we add the prototypes (or an example) here so the users don't need to look in the header file?

Reply:
Good, I added the prototype as it is done for most other commands.